src/utils/ImagePicker.ts

```diff
@@ -0,0 +1,35 @@
+import * as ImagePicker from 'expo-image-picker';
+import * as Permissions from 'expo-permissions';
+
+const photoOptions = {
+  mediaTypes: ImagePicker.MediaTypeOptions.Images,
+  allowsEditing: true,
+  exif: true,
+};
+
+const requestPermissions = async (
```

Comment:
To test the below lines, you may need to mock `Permissions` module or `permissions.askAsync` function.

Reply:
resolved!